src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,822 @@
+  // force thumb to have focus
+  thumb.focus();
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('50');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowLeft' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('50');
+});
+
+it('should show tooltip on thumb hover', () => {
+  const wrapper = render(<Slider values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+  expect(container.querySelector('.iui-tooltip')).toBeFalsy();
+  act(() => {
+    fireEvent.mouseEnter(thumb);
+  });
+  expect(
+    (container.querySelector('.iui-tooltip') as HTMLDivElement).textContent,
+  ).toBe('50');
+  act(() => {
+    thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+    fireEvent.mouseLeave(thumb);
+  });
+});
+
+it('should show tooltip on thumb focus', async () => {
+  const wrapper = render(<Slider values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+  expect(container.querySelector('.iui-tooltip')).toBeFalsy();
+  act(() => {
+    thumb.focus();
+  });
+  expect(
+    (container.querySelector('.iui-tooltip') as HTMLDivElement).textContent,
+  ).toBe('50');
+
+  await act(async () => {
+    thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+    thumb.blur();
+  });
+  // The tooltip, which was constructed earlier in test, remains but it is set to hidden.
+  const tippyBox = container.querySelector('.tippy-box') as HTMLDivElement;
+  expect((tippyBox.parentElement as HTMLElement).style.visibility).toEqual(
+    'hidden',
+  );
+});
+
+it('should apply thumb props', () => {
+  const thumbProps = () => {
+    return {
+      className: 'thumb-test-class',
+      style: { backgroundColor: 'red' },
+    };
+  };
+  const wrapper = render(
+    <Slider values={defaultSingleValue} thumbProps={thumbProps} />,
+  );
+  const { container } = wrapper;
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('thumb-test-class'));
+  expect(thumb.style.backgroundColor).toEqual('red');
+});
+
+it('should focus on specific thumb', () => {
+  const wrapper = render(<Slider values={[50, 80]} step={5} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  const sliderContainer = container.querySelector(
+    '.iui-slider-container',
+  ) as HTMLDivElement;
+  let thumbs = container.querySelectorAll('.iui-slider-thumb');
+
+  focusThumb(sliderContainer, 1);
```

Comment:
As these are tests, would be better to just find the thumb you want and focus it through code.. so we do not export inner function for it..

Reply:
I was using this prior to setting PointerEvent to MouseEvent in test... it can be changed.